columnflow/config_util.py

```diff
@@ -202,15 +205,16 @@ def kwargs_fn(categories):
             for root_cats in itertools.product(*_categories):
                 # build the name
                 root_cats = dict(zip(_group_names, root_cats))
-                cat_name = name_fn(**{
-                    group_name: cat.name
-                    for group_name, cat in root_cats.items()
-                })
+                cat_name = name_fn(root_cats)
 
                 # skip when already existing
                 if skip_existing and config.has_category(cat_name, deep=True):
                     continue
 
+                # skip when manually triggered
+                if callable(skip_fn) and skip_fn(root_cats, cat_name):
```

Comment:
Since `name_fn` now received only `root_cats`, is there a reason for the additional `cat_name` being passed to `skip_fn` here?

Reply:
Right ... even in the example above I skipped it ^^. Let's remove it.